packages/vuetify/src/composables/elevation.ts

```diff
@@ -25,13 +26,18 @@ export const makeElevationProps = propsFactory({
   },
 }, 'elevation')
 
-export function useElevation (props: ElevationProps) {
+type ElevationData = {
+  elevationClasses: Ref<string[]>
+}
+
+export function useElevation (props: ElevationProps | Ref<number | string | undefined>): ElevationData {
   const elevationClasses = computed(() => {
+    const elevation = isRef(props) ? props.value : props.elevation
```

Comment:
This shouldn't be here tbh. Why not handle this within the implementing component?

```ts
const { elevationClasses } = useElevation(computed(() => {
  return { elevation: props.value || props.elevation }
})
```

Reply:
That doesn't work without this change.